src/web/aws-cloudwatch/FormAdvancedOptions.jsx

```diff
@@ -14,32 +18,32 @@ const FormAdvancedOptions = ({ onChange, values, toggle, visible }) => {
         <Input id="awsCloudWatchGlobalInput"
                type="checkbox"
                value="global-input"
-               defaultChecked={values.awsCloudWatchGlobalInput}
+               defaultChecked={getFieldValue('awsCloudWatchGlobalInput')}
```

Comment:
Is it possible to turn this into a object returned by `formDataHook` that we can destructure, so we do not have to do a function call for every input?

Reply:
Hopefully you have a better idea for me, because this was the cleanest I could make it.
If I replace the functions, then every value ends up being `formData.awsCloudWatchName.value || formData.awsCloudWatchName.defaultValue`